metaworld/envs/mujoco/sawyer_xyz/sawyer_peg_insertion_side_v2.py

```diff
@@ -104,24 +104,17 @@ def _set_obj_xyz(self, pos):
 
     def reset_model(self):
         self._reset_hand()
-        self.sim.model.body_pos[self.model.body_name2id('box')] = self.goal.copy()
+
+        self.sim.model.body_pos[self.model.body_name2id('box')] = np.array([-0.3, 0.6, 0.05])
         self._state_goal = self.sim.model.site_pos[self.model.site_name2id('hole')] + self.sim.model.body_pos[self.model.body_name2id('box')]
         self.obj_init_pos = self.init_config['obj_init_pos']
         self.objHeight = self.get_body_com('peg').copy()[2]
         self.heightTarget = self.objHeight + self.liftThresh
 
         if self.random_init:
```

Comment:
maybe i missed this in previous PRs, but wasn't `random_init` completely removed?

Reply:
It was removed as an arg to the constructor, but still exists as an attribute of the classes. It's just always set to True.